package.json

```diff
@@ -64,7 +64,7 @@
   "peerDependencies": {
     "react": "^18.0.0 || ^17.0.0 || ^16.8.0",
     "react-dom": "^18.0.0 || ^17.0.0 || ^16.8.0",
-    "stream-chat": "^7.1.0"
+    "stream-chat": "^8.0.0 || ^7.1.0"
```

Comment:
```suggestion
    "stream-chat": ">= 7.1.0"
```

Reply:
Or whatever oldest version we decide to support.

Reply:
I think we should stay in the range that @MartinCupela proposed for the time being.
I'm not sure how safe would it be if we have, for example, `chat@10.4` and `client@12`.